src/ci/docker/scripts/fuchsia-test-runner.py

```diff
@@ -849,18 +848,14 @@ def debug(self, args):
         ]
 
         # Add rust source if it's available
+        rust_src_map = None
         if args.rust_src is not None:
-            command += [
-                "--build-dir",
-                args.rust_src,
-            ]
+            rust_src_map = f"/rustc/FAKE_PREFIX={args.rust_src}"
```

Comment:
Add a comment saying this duplicates the prefix used by compiletest (a bit unfortunate but not obvious how to fix)